features/images.feature

```diff
@@ -0,0 +1,40 @@
+@images
+Feature: OpenStack Image management
+  As an administrator
+  I want to make sure that OpenStack Image management component
+  is working correctly
+
+  @create_kvm
+  Scenario: Creation of KVM image
+    Given KVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos
+    Then this image has non-empty ID
+    And its ID can be used to show the image info
+    And the status of the image is active
+    When I delete the image
```

Comment:
> I'm not sure. All scenarios are creating different images, so I'm not sure why should I have one scenario for deleting all of them.

You shouldn't.  Noone's proposing that.

> And also, I like it more when the situation is back to original state after executing each scenario, not just after running full feature.

Yes, that's what I'm proposing.  But my suggestion avoids polluting the image *creation* test with that cleanup.

> Background is nice, yes, e.g. for testing something that is really shared

No, background is not for shared tests, it's for shared setup (context).  As I already showed in the discussion linked above, image cleanup can be part of the setup. 

> (and for 2 creating scenarios, they have different image id and name)

Sure, and as I already said, this is easily handled via `@image_id` and `@image_name`.

> and I wouldn't expect it to delete the image if it exist

If the background has a step `Given glance image does not exist` then why wouldn't you expect it to delete any existing glance image?

Sorry, but it sounds like you still don't understand what I'm proposing.  Maybe reading the linked discussion will help?